dockerfiles/docker/Dockerfile

```diff
@@ -5,11 +5,9 @@ RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-ada
 
 FROM rcjsuen/docker-langserver
 
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
-ENTRYPOINT ["/tini", "--"]
+RUN apk add --no-cache tini
+ENTRYPOINT ["tini", "--"]
```

Comment:
It seems if the base image uses alpine the downloading of tini does not work. I assume maybe some shard library is missing? Either way switching to apk works fine :)

Reply:
I prefer the apk method when we are using alpine anyway.

Reply:
Yeah, I had to do this for our other alpine images. I still think it's fine to use this method in the example Dockerfile since we can't assume they're using alpine